ProjectLighthouse/Database.cs

```diff
@@ -312,6 +317,40 @@ public async Task UnqueueLevel(int userId, Slot queuedSlot)
         await this.SaveChangesAsync();
     }
 
+    public async Task BlockUser(int userId, User blockedUser)
+    {
+        if (userId == blockedUser.UserId)
+        {
+            return;
+        }
+        
+        User? user = await this.Users.FirstOrDefaultAsync(u => u.UserId == userId);
+        
+        BlockedProfile blockedProfile = new()
+        {
+            User = user,
+            BlockedUser = blockedUser,
```

Comment:
Does this properly set the UserId and BlockedUserId fields? Generally, I like to set the actual fields that will update in the db but if it works then I guess it's fine.

Reply:
It worked in my testing